File: src/migratoryBirds.scala
    def migratoryBirds(arr: Array[Int]): Int = {
        val srtd = arr.sorted 
        val counts = srtd.distinct.map(x => { (x, srtd.filter(_ == x).size) } )
        val maxCount = counts.map(p => p._2).max
        val candidates = counts.filter(p => {p._2 == maxCount})
        val result = candidates.map(c => c._1).min
        return result
    }

File: src/migratoryBirds.py
def migratoryBirds(arr):
    '''
    Return most frequent integer in array
    If tie, return min
    '''
    counter = {}
    key_set = set(arr)
    for k in key_set:
        counter[k] = arr.count(k)
    candidates = [k for k in counter.keys() if counter[k] == max(counter.values())]
    return min(candidates)
